pyproject.toml

```
@@ -45,12 +45,8 @@ dependencies = [
     "types-pyyaml>=6.0.12.20240917,<7",
     "pytest-json-report>=1.5.0,<2",
     "typing-extensions>=4.12.2,<5",
-    # TODO: bump questionary to a newer release, when it becomes available.
-    # The current release questionary 2.0.1 requires `prompt_toolkit = ">=2.0,<=3.0.36"`.
-    # This conflicts with ipython; while not an EEST dependency, ipython is a very useful tool:
-    # https://eest.ethereum.org/main/dev/interactive_usage/
-    "questionary @ git+https://github.com/tmbo/questionary@ff22aeae1cd9c1c734f14329934e349bec7873bc",
-    "prompt_toolkit>=3.0.48,<4", # ensure we have a new enough version for ipython
+    "questionary>=2.1.0",
```

Although we lock packages to an explicit version when we use EEST internally (locally, in editable mode), this is not the case when we publish EEST as a package so I think it's best to protect against breaking changes in new major versions by capping the max version:
```suggestion
    "questionary>=2.1.0,<3",
```

Please don't accept the suggest, but rather run:
`uv add "questionary>=2.1.0,<3"`